diff --git a/luci-app-passwall/Makefile b/luci-app-passwall/Makefile
--- a/luci-app-passwall/Makefile
+++ b/luci-app-passwall/Makefile
@@ -66,4 +66,4 @@
 config PACKAGE_$(PKG_NAME)_INCLUDE_Geoview
 	bool "Include Geoview"
 	select PACKAGE_geoview
-	default y if aarch64||arm||i386||x86_64
+	default n
@@ -121,4 +121,4 @@
 config PACKAGE_$(PKG_NAME)_INCLUDE_SingBox
 	bool "Include Sing-Box"
 	select PACKAGE_sing-box
-	default y if aarch64||arm||i386||x86_64
+	default n
